src/store.ts

```diff
@@ -69,6 +71,43 @@ export const DefaultMarkThresholds: Readonly<MarkThresholds> = {
   definite: 0.25,
 }
 
+const leanInterpretation = (
+  interpretation: PageInterpretation | undefined
+): PageInterpretation | undefined => {
+  if (!interpretation) {
+    return interpretation
+  }
+
+  if (interpretation.type !== 'InterpretedHmpbPage') {
+    return { ...interpretation }
```

Comment:
Any reason not to just do this?

```suggestion
    return interpretation
```

Reply:
I'm just getting super paranoid about mutability :)